django/db/models/sql/compiler.py

```diff
@@ -1084,8 +1084,10 @@ def as_sql(self):
             return '', ()
         table = self.query.tables[0]
         qn = self.quote_name_unless_alias
-        result = ['UPDATE %s' % qn(table)]
-        result.append('SET')
+        result = [
+            'UPDATE %s' % qn(table),
+            'SET',
```

Comment:
I think 'SET' can be included in the first string (at least the tests pass on SQLite with this change)?